### ISSUE TITLE
requirement.txt

### ISSUE THREAD

**Issue by ZBoIsHere** [ZBoIsHere/ZBoIsHere#11]
~orch==1.11.0+cu113
~pencv_contrib_python==4.8.0.76
numpy==1.24.3
Pillow==10.2.0
segment_anything==1.0
asposestorage==1.0.2
labels==20.1.0
torch==1.11.0+cu113
train==0.0.5
utils==1.0.2
cv_bridge==1.13.0.post0
datasets==2.18.0
glfw==2.7.0
groundingdino.egg==info
h5py==3.9.0
imageio==2.34.0
kaolin.egg==info
kitti360scripts==1.0.0
kornia==0.6.12
lietorch==0.6.2.post1
lpips==0.1.4
matplotlib==3.6.1
mmdet3d==1.4.0
mmsegmentation.egg==info
nvdiffrast==0.3.1
open3d==0.17.0
opencv_contrib_python_headless==4.7.0.72
opencv_python==4.8.0.76
opencv_python_headless==4.8.0.76
packaging==24.0
pandas==1.5.3
pyfqmr==0.2.0
PyMCubes==0.1.4
pymeshlab==2022.2.post4
PyOpenGL==3.1.7
PyQt5==5.15.10
PyQt5_sip==12.12.1
pyrender==0.1.45
pytorch3d==0.3.0
PyYAML==6.0.1
pyzmq==25.1.0
scikit_learn==1.2.2
scipy==1.9.1
sdf==0.3.5
segment_anything==1.0
setuptools==68.2.2
shencoder.egg==info
sip==6.8.3
skimage==0.0
smooth_sampler==0.0.1
stl==0.0.3
tifffile==2024.2.12
torchvision==0.12.0+cu113
tqdm==4.65.0
trimesh==3.22.5
wisp.egg==info
